tools/cpp_build/build_caffe2.sh

```diff
@@ -1,33 +0,0 @@
-#!/usr/bin/env bash
-
-set -ex
-
-SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
-
-pushd $SCRIPTPATH
-source ./build_common.sh
-
-echo "Building Caffe2"
-
-mkdir -p $CAFFE2_BUILDPATH
-pushd $CAFFE2_BUILDPATH
-
-cmake -DUSE_CUDA:BOOL=$USE_CUDA \
-      -DBUILD_TORCH=ON \
-      -DUSE_OPENMP:BOOL=${USE_OPENMP:ON} \
```

Comment:
Will `USE_OPENMP` still work?

Reply:
yes, though when you're using the build_pytorch_libs it will look like

```
EXTRA_CAFFE2_CMAKE_FLAGS="USE_OPENMP=1" bash tools/build_pytorch_libs.sh ...
```

which is a bit ugly, but it does work and we are already separately working on making build_pytorch_libs "nicer"